WeakAurasOptions/SubRegionOptions/SubText.lua

```diff
@@ -218,7 +218,54 @@ local function createOptions(parentData, data, index, subIndex)
       values = WeakAuras.justify_types,
       order = 50.5,
       hidden = hiddenFontExtra
-    }
+    },
+    text_font_space5 = {
+      type = "description",
+      name = "",
+      order = 51,
+      hidden = hiddenFontExtra,
+      width = indentWidth
+    },
+    text_automaticWidth = {
+      type = "select",
+      width = WeakAuras.normalWidth - indentWidth,
+      name = L["Width"],
+      order = 51.5,
+      values = WeakAuras.text_automatic_width,
+      hidden = hiddenFontExtra
+    },
+    text_font_space6 = {
+      type = "description",
+      name = "",
+      order = 52,
+      hidden = hiddenFontExtra,
+      width = WeakAuras.normalWidth
+    },
+    text_font_space7 = {
+      type = "description",
+      name = "",
+      order = 52.5,
+      width = indentWidth,
+      hidden = function() return hiddenFontExtra() or data.text_automaticWidth  ~= "Fixed" end,
+    },
+    text_fixedWidth = {
+      name = L["Width"],
+      width = WeakAuras.normalWidth - indentWidth,
+      order = 53,
+      type = "range",
+      min = 1,
+      softMax = 200,
+      bigStep = 1,
+      hidden =  function() return hiddenFontExtra() or data.text_automaticWidth  ~= "Fixed" end
+    },
+    text_wordWrap = {
+      type = "select",
+      width = WeakAuras.normalWidth,
+      name = L["Overflow"],
+      order = 54,
+      values = WeakAuras.text_word_wrap,
+      hidden = function() return hiddenFontExtra() or data.text_automaticWidth  ~= "Fixed" end
+    },
```

Comment:
The text_fontFlagsDescription option should mention if fixedWidth/text_wordWrap is set, so that it is visible even in the while the options are collapsed.

Otherwise this looks good to me.